src/usage.ts

```diff
@@ -91,7 +91,8 @@ export async function renderUsage<T extends ArgsDef = ArgsDef>(
     "",
   );
 
-  const hasOptions = argLines.length > 0 || posLines.length > 0;
```

Comment:
I'm not sure to know to what `posLines` is referring here.

Reply:
positional args

Reply:
thanks!